src/main/java/com/google/sps/servlets/RecommendationServlet.java

```diff
@@ -64,11 +50,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       Map<String, String[]> jsonInfo = new HashMap<>();
       jsonInfo.put(queryWord, requestedNeighbors);
+      response.setContentType("application/json;");
       response.getWriter().println(new Gson().toJson(jsonInfo));
     } catch (NullPointerException e) {
-      Map<String, String> jsonErrorInfo;
-      jsonErrorInfo = ResponseSerializer.getErrorJson("Cannot find similar words at the moment");
-      response.getWriter().println(new Gson().toJson(jsonErrorInfo));
+      ResponseSerializer.sendErrorJson(response, "Cannot find similar words at the moment");
```

Comment:
"at the moment" could imply if the user waits a moment, there is a chance they'll get a response with similar words. Maybe keep the response to "Cannot find similar word."

Reply:
done